### DISCUSSION TITLE
Is it possible to set session timeout?

### DISCUSSION THREAD

**Discussion by hellosmriti** [pimcore/pimcore#7834] [General]
The application admin doesn't terminate the session automatically even if the user session is inactive for a prolonged period after login.

Is it possible to set session timeout?

**Comment by NiklasBr** [comment]
According to [Security and Authentication docs](https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Security_Authentication/index.html) Symfony Sessions are used, so you could try using [session.cookie_lifetime](https://symfony.com/doc/current/reference/configuration/framework.html#cookie-lifetime) to control it.

**Reply by hellosmriti** [reply]
> According to [Security and Authentication docs](https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Security_Authentication/index.html) Symfony Sessions are used, so you could try using [session.cookie_lifetime](https://symfony.com/doc/current/reference/configuration/framework.html#cookie-lifetime) to control it.

Thanks @NiklasBr.
